lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -161,11 +161,22 @@ defmodule ExUnit.Runner do
     running
   end
 
-  defp spawn_modules(config, [{module, params} | modules], async?, running) do
+  defp spawn_modules(
+         config,
+         [{_group, group_modules} | modules],
+         async?,
+         running
+       ) do
     if max_failures_reached?(config) do
       running
     else
-      {pid, ref} = spawn_monitor(fn -> run_module(config, module, async?, params) end)
+      {pid, ref} =
+        spawn_monitor(fn ->
+          Enum.each(group_modules, fn {module, params} ->
```

Comment:
I believe this will run the `setup_all` of all modules in a given group in the same process and we likely want to have a different process for each of them instead... 🤔

Reply:
I am wrong. We spawn a separate process later on. :)